File: Java/Basic/basicLevel1038/main.cpp
#include <iostream>

using namespace std;

int main() {
    int *score = new int [101];
    int n = 0;
    cin >> n;
    
    for (int i = 0; i < n; i++) {
        int temp = 0;
        cin >> temp;
        score[temp]++;
    }
    int k = 0;
    cin >> k;
    for (int i = 0; i < k-1; i++) {
        int temp = 0;
        cin >> temp;
        cout << score[temp] << " ";
    }
    int temp = 0;
    cin >> temp;
    cout << score[temp];

    delete [] score;
    
}

File: Java/Basic/basicLevel1038/Main.java
package basicLevel1038;

// The test point four is runtime is outed.
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {

		int[] score = new int [101];
		Scanner in = new Scanner(System.in);
		int n = in.nextInt();
		in.nextLine();
		
		
		String[] input = in.nextLine().split(" ");
		
		for (int i = 0; i < n; i++) {
			int temp = Integer.parseInt(input[i]);
			score[temp]++;
		}
		int k = in.nextInt();
		for (int i = 0; i < k-1; i++) {
			int temp = in.nextInt();
			System.out.print(score[temp] + " ");
		}
		int tmep = in.nextInt();
		System.out.print(score[tmep]);
		in.close();
	}

}
